firmware/config/boards/subaru_eg33/board_configuration.cpp

```diff
@@ -262,3 +263,97 @@ void setBoardDefaultConfiguration(void) {
 	if (engineConfiguration->fuelAlgorithm == LM_ALPHA_N)
 		setAlgorithm(LM_ALPHA_N PASS_CONFIG_PARAMETER_SUFFIX);
 }
+
+static const struct mc33810_config mc33810_odd = {
+	.spi_bus = &SPID5,
+	.spi_config = {
+		.circular = false,
+		.end_cb = NULL,
+		.ssport = GPIOF,
+		.sspad = 1,
+		.cr1 =
+			//SPI_CR1_16BIT_MODE |
+			SPI_CR1_SSM |
+			SPI_CR1_SSI |
+			((3 << SPI_CR1_BR_Pos) & SPI_CR1_BR) |	/* div = 16 */
+			SPI_CR1_MSTR |
+			/* SPI_CR1_CPOL | */ // = 0
+			SPI_CR1_CPHA | // = 1
+			0,
+		.cr2 = //SPI_CR2_16BIT_MODE |
+			SPI_CR2_DS_3 | SPI_CR2_DS_2 | SPI_CR2_DS_1 | SPI_CR2_DS_0
+	},
+	.direct_io = {
+		/* injector drivers */
+		[0] = {.port = GPIOI, .pad = 6},	/* INJ 1 */
+		[1] = {.port = GPIOI, .pad = 5},	/* INJ 3 */
+		[2] = {.port = GPIOI, .pad = 4},	/* INJ 5 */
+		[3] = {.port = GPIOB, .pad = 9},	/* INJ 7 */
+		/* ignition pre-dirvers */
+		[4] = {.port = GPIOB, .pad = 3},	/* IGN 1 */
+		[5] = {.port = GPIOB, .pad = 4},	/* IGN 3 */
+		[6] = {.port = GPIOB, .pad = 5},	/* IGN 7 */
+		[7] = {.port = GPIOB, .pad = 8},	/* IGN 5 */
+	},
+	/* en shared between two chips */
+	.en = {.port = GPIOI, .pad = 7}
+};
+
+static const struct mc33810_config mc33810_even = {
+	.spi_bus = &SPID5,
+	.spi_config = {
+		.circular = false,
+		.end_cb = NULL,
+		.ssport = GPIOF,
+		.sspad = 2,
+		.cr1 =
+			//SPI_CR1_16BIT_MODE |
+			SPI_CR1_SSM |
+			SPI_CR1_SSI |
+			((3 << SPI_CR1_BR_Pos) & SPI_CR1_BR) |	/* div = 16 */
+			SPI_CR1_MSTR |
+			/* SPI_CR1_CPOL | */ // = 0
+			SPI_CR1_CPHA | // = 1
+			0,
+		.cr2 = //SPI_CR2_16BIT_MODE |
+			SPI_CR2_DS_3 | SPI_CR2_DS_2 | SPI_CR2_DS_1 | SPI_CR2_DS_0
+	},
+	.direct_io = {
+		/* injector drivers */
+		[0] = {.port = GPIOE, .pad = 3},	/* INJ 2 */
+		[1] = {.port = GPIOE, .pad = 4},	/* INJ 4 */
+		[2] = {.port = GPIOE, .pad = 5},	/* INJ 6 */
+		[3] = {.port = GPIOE, .pad = 6},	/* INJ 8 */
+		/* ignition pre-dirvers */
+		[4] = {.port = GPIOC, .pad = 14},	/* IGN 2 */
+		[5] = {.port = GPIOC, .pad = 13},	/* IGN 4 */
+		[6] = {.port = GPIOC, .pad = 15},	/* IGN 6 */
+		[7] = {.port = GPIOI, .pad = 9},	/* IGN 8 */
+	},
+	/* en shared between two chips */
+	// TODO: dron0gus, what to do here?
+	.en = {.port = GPIOI, .pad = 7}
```

Comment:
@dron0gus what to do about this one? C++ requires that all fields are initialized, so should we explicitly init this as no-pin or something?